mne/datasets/utils.py

```diff
@@ -375,6 +395,14 @@ def _download_all_example_data(verbose=True):
     limo.load_data(subject=1, update_path=True)
     logger.info("[done limo]")
 
+    # for ESG
+    ds = "ds004388"
+    target_dir = default_path() / ds
+    run_name = "sub-001/eeg/*median_run-03_eeg*.set"
+    if not glob.glob(str(target_dir / run_name)):
+        target_dir.mkdir(exist_ok=True)
+        openneuro.download(dataset=ds, target_dir=target_dir, include=run_name[:-4])
```

Comment:
@larsoner this code is duplicated in the example (which it wouldn't need to be if this were a full-fledged MNE dataset). Is the reason for not making this a full-fledged dataset that you don't expect ESG data to be used / useful in other tutorials?

Reply:
We could add it as a new dataset in theory. But it would be nice not to have to do that for stuff that's on OpenNeuro already. So I'm thinking this could be a new way to add data when people want.

What's here at least works. I suggest we see how much we end up using it and doing things similarly in the future and eventually simplify / add helpers as needed. I think it will make sense in the end to download to `mne_data/ds000XYZW` at the end of the day just like `mne_bids_pipeline` does already, so no wasted storage at the users end in the meantime.

Reply:
that reasoning makes sense to me. So now just need to decide which `project.optional-dependencies` group in `pyproject.toml` the new `openneuro` dependency goes in.  `data` group seems like a natural fit, but it's an empty list kept around for backward compatibility :/ and IIRC was where HDF5 stuff used to live.  WDYT?

Reply:
I think `docs` should be good enough, I'll push that